docs/associations/belongs-to-many.md

```diff
@@ -0,0 +1,468 @@
+---
+sidebar_position: 5
+title: BelongsToMany
+---
+
+# The BelongsToMany Association
+
+The BelongsToMany association is used to create a [Many-To-Many relationship](https://en.wikipedia.org/wiki/Many-to-many_(data_model)) between two models.
+
+In a Many-To-Many relationship, a row of one table is associated with _zero, one or more_ rows of another table, and vice versa.
+
+For instance, a person can have liked zero or more Toots, and a Toot can have been liked by zero or more people.
+
+```mermaid
+erDiagram
+  people }o--o{ toots : likedToots
+```
+
+Because foreign keys can only point to a single row, Many-To-Many relationships are implemented using a junction table (called __through table__ in Sequelize), and are 
+really just two One-To-Many relationships.
+
+```mermaid
+erDiagram
+  people }o--|| liked_toots : user
+  liked_toots ||--o{ toots : toot
+```
+
+The junction table is used to store the foreign keys of the two associated models.
+
+## Defining the Association
+
+Here is how you would define the `Person` and `Toot` models in Sequelize:
+
+```ts
+import { Model, InferAttributes, InferCreationAttributes, NonAttribute } from '@sequelize/core';
+import { BelongsToMany } from '@sequelize/core/decorators-legacy';
+
+class Person extends Model<InferAttributes<Person>, InferCreationAttributes<Person>> {
+  // highlight-start
+  @BelongsToMany(() => Toot, {
+    through: 'LikedToot',
+  })
+  declare likedToots?: NonAttribute<Toot[]>;
+  // highlight-end
+}
+
+class Toot extends Model<InferAttributes<Toot>, InferCreationAttributes<Toot>> {}
+```
+
+In the example above, the `Person` model has a Many-To-Many relationship with the `Toot` model, using the `LikedToot` junction model.
+
+The `LikedToot` model is automatically generated by Sequelize, and will receive the two foreign keys: `userId` and `tootId`.
+
+:::caution `through` option
+
+The `through` option is not a table name but a model name. Its corresponding table name will be generated by Sequelize, and
```

Comment:
I don't really understand this sentence. Do you mean that you need to provide the model name under `through` instead of the table name?

Reply:
If you have a model called "UserProject" that maps to a table called "user_projects", you need to use `UserProject` here

I've seen many, many people think they need to provide the name of the through table, when it's the name of the through model

Reply:
Updated wording